README.md

```diff
@@ -1,35 +1,39 @@
 # DACSspace
 
-A simple Python script to evaluate your ArchivesSpace instance for DACS compliance.
-
-This repository is currently under development!
+A simple Python script to evaluate your ArchivesSpace instance for DACS [single-level minimum](http://www2.archivists.org/standards/DACS/part_I/chapter_1) required elements. 
```

Comment:
This could include more info, like mentioning using the AS API + that it produces a CSV (& what's in the CSV)